bitnami/thanos/README.md

```diff
@@ -131,7 +131,14 @@ The following tables lists the configurable parameters of the Thanos chart and t
 | `querier.podAnnotations`                        | Annotations for Thanos Querier pods                                                                    | `{}`                                                    |
 | `querier.livenessProbe`                         | Liveness probe configuration for Thanos Querier                                                        | `Check values.yaml file`                                |
 | `querier.readinessProbe`                        | Readiness probe configuration for Thanos Querier                                                       | `Check values.yaml file`                                |
-| `querier.service.type`                          | Kubernetes service type                                                                                | `ClusterIP`                                             |
+| `querier.grpcTLS.server.cert`                   | TLS Certificate for gRPC server                                                                        | `nil`                                                   |
+| `querier.grpcTLS.server.key`                    | TLS Key for gRPC server                                                                                | `nil`                                                   |
+| `querier.grpcTLS.server.clientCA`               | TLS clientCA for gRPC server.If no client CA is specified, there is no client verification on server.  | `nil`                                                   |
+| `querier.grpcTLS.client.secure`                 | Use TLS when talking to the gRPC server                                                                | `false`                                                 |
```

Comment:
I would add a secure parameter like this one to the server configuration too to be consistent